src/Caliburn.Micro.WinUI3/Caliburn.Micro.WinUI3.csproj

```diff
@@ -0,0 +1,72 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <RootNamespace>Caliburn.Micro</RootNamespace>
+    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
```

Comment:
could you change this to 
    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>